keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/README.md

```diff
@@ -609,8 +612,7 @@ _This section must be completed when targeting beta graduation to a release._
   is very limited and it is disabled by default.
 
 * **What specific metrics should inform a rollback?**
-  We could have a metric in Kubelet that records number of failed pods that use this feature. If that
-  metric spikes we could trigger a rollback.
+  Abnormal increase in `run_podsandbox_errors_total` count could be related to this feature. We should check if the feature gate is enabled and pods are using it.
```

Comment:
L611: it's no longer disabled by default in Beta.

I agree that I don't see any failure mode during rollout though - let's make it clear:
"No known failure modes"

Reply:
This line: this is about rollout, so we know the feature is enabled :)
So I guess what you're saying is that we should list pods, filter those using this feature and check whether they are not stuck in the state described in the proposal, right?

Reply:
nit: L618: let's use enabling/disabling the feature (instead of introducing and removing the feature) [wording]